### ISSUE TITLE
Blender 2.80.0 80ffa73b3f87 - import issue.

### ISSUE THREAD

**Issue by moisoni97** [Befzz/blender3d_import_psk_psa#22]
When trying to import in blender version mentioned in title I got the following error: `AttributeError: 'RNA_Tyoes' object has no attribute 'INFO_MT_file_import`. However in blender 2.7b it works.

**Comment by Befzz** [comment]
i have just checked `January 11, 18:25:52 - 1c7695b8483d 2.8 build`   
Are you using addon from **"latest"** branch?:  
https://github.com/Befzz/blender3d_import_psk_psa/tree/latest/addons

**Comment by moisoni97** [comment]
No, I just downloaded the script from the main page a few days ago. Now it's working.
Thanks!
